apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Trace/TraceTest.cs

```diff
@@ -193,24 +199,27 @@ public class TraceTest
             string testId = Utils.GetTestId();
             var startTime = Timestamp.FromDateTime(DateTime.UtcNow);
 
-            var client = _noBufferHighQps.CreateClient();
-            try
+            using (var server = new TestServer(new WebHostBuilder().UseStartup<TraceTestNoBufferHighQpsApplication>()))
+            using (var client = server.CreateClient())
             {
-                await client.GetAsync($"/Trace/ThrowException/{testId}");
+                try
+                {
+                    await client.GetAsync($"/Trace/ThrowException/{testId}");
```

Comment:
Not that its something you added here, but when an exception is expected, it's probably a good idea to assert if you get past this line using `Assert.True(false, ...)`